src/properties.js

```diff
@@ -12,115 +122,30 @@ const enrichPropertyDefinitions = (propertyDefinitions) => {
 
       return propertyMap[key] || key;
     },
-    formatPropertyValue: (key, value) => {
-      let suffix = "";
-
-      if (typeof propertyMap[key] === "object") {
-        if (propertyMap[key].suffix != null) {
-          suffix = propertyMap[key].suffix;
-        }
-
-        if (propertyMap[key].type === "date") {
-          return new Date(value).toLocaleString([], {
-              timeZone: "UTC",
-              timeZoneName: "short"
-            }) + suffix;
+    formatSummaryValues: (key, values) => {
+      if (Array.isArray(values)) {
```

Comment:
What are you thoughts about making this the format for all list values? Is there a reason to have summary and other metadata lists in the different formats?

Reply:
Yes, except the sorting.